scripts/Two_VM_Deployment/create-kubernetes-cluster.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -x
```

Comment:
Are these file copied from scripts/terraform?

Reply:
They are similar in some parts to files scripts/terraform, particularly install-kubernetes (in some parts), and download-worker-images.sh.
But the other files are different. the script create-kubernetes-cluster.sh keeps the same name, but the content is different. I
Do you suggest to change the name, or to remove the similarities? 
In fact, scripts in scripts/terraform or scripts/vagrant can not be used alone, in their current form, to deploy a non-vagrant 2-VMs K8s cluster (master+worker). That's the main insight of scripts/Two_VM_Deployment.

Reply:
I would prefer to have a single set of scripts that can initialize *any* cluster for us. The script itself can get arguments when called so it modifies its behaviour accordingly.

Reply:
Ok, so do you propose that I create a new set of scripts for both vagrant and non-vagrant k8s clusters? or to modify the current scripts/vagrant to do that?